### PR TITLE
Hotfix/2.2.0

### PR DESCRIPTION
**Pull request by CiaranOMara**
https://github.com/BioJulia/BioAlignments.jl/pull/72#issuecomment-1147482102

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
+
+### Removed
+- :exclamation: Reverted the use of *BioJulia* registry,
+  the package switched to [General Julia Registry](https://github.com/JuliaRegistries/General) (#48)
+- :exclamation: Removed Gitter chat links (#ebea200c)
+- :exclamation: Dropped support for julia less than v1.6 (#72)
+
+### Fixed
+- Doctests (#78)
+
```
Comment (MillironX) [inline CHANGELOG.md:36]: Let's add a `## 0.2.1 - 2022-10-22 [YANKED]` section (<https://keepachangelog.com/en/1.0.0/#yanked>) to include the merging of #44 and c49bb0a3. We'll note that they are re-included in the 3.0 section.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,39 @@
...
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Added
+- Alignment position support (#44)
```
Comment (MillironX) [inline CHANGELOG.md:11]: Since the whole point of this hotfix is to revert this change, it shouldn't be in the changelog.

Comment (CiaranOMara) [reply CHANGELOG.md:11]: Good catch.

File: CHANGELOG.md (hunk 1 above)
Comment (CiaranOMara) [reply CHANGELOG.md:40]: I've Included the changelog entry. Let me know if the entry should be in a different spot and/or if other information needs adding to it.

The Zenodo badge is included in this hotfix.

**Comment by CiaranOMara** [comment]
I've converted this PR to a draft to get feedback on merging #60 and including it here. I think #60 is the last outstanding non-breaking feature request.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,45 @@
...
+  the package switched to [General Julia Registry](https://github.com/JuliaRegistries/General) (#48)
+- :exclamation: Removed Gitter chat links (#ebea200c)
+- :exclamation: Dropped support for julia less than v1.6 (#72)
+
+### Fixed
+- Doctests (#78)
+
+## 0.2.1 - 2022-10-22 [YANKED]
+
 ## [2.0.0]
```
Comment (MillironX) [inline CHANGELOG.md:43]:
```suggestion

## [Unreleased]

## [2.2.0]

### Added
- Compact printing of alignments (#53)
- Downstream tests (#70)
- SSH secret to TagBot.yml (#71)
- Doctests as part of the unit tests (#78)

### Changed
- Updated chat links (#ebea200c)
- Expanded test matrix, adding julia v1.6 (#77)
- Incremented BioSequences compatibility to v3 (#72)
- Incremented BioSymbols compatibility to v5 (#72)
- Adjusted test matrix for minimum compatibility (#72)
- Updated doctests (#72)

### Removed
- :exclamation: Removed Gitter chat links (#ebea200c)
- :exclamation: Dropped support for julia less than v1.6 (#72)
- :exclamation: Removed alignment position support, as it was a breaking change (#44, #76)

### Fixed
- Doctests (#78)

## [2.1.0] - 2022-02-10 [YANKED]

### Added
- Zenodo badge (#c49bb0a3)

### Removed
- :exclamation: Dropped support for julia less than v1 (#50)

### Changed

## [2.0.1] - 2022-02-08 [UNREGISTERED]

### Added
- Alignment position support (#44)

### Changed
- Updated CI to use GitHub actions (#47)
- *master* set as the main development branch (#49, #50)

### Removed
- :exclamation: Reverted the use of *BioJulia* registry,
  the package switched to [General Julia Registry](https://github.com/JuliaRegistries/General) (#48)

## [2.0.0]
```

I think this is best way to reflect the state of the project.

Comment (CiaranOMara) [reply CHANGELOG.md:43]: That's a better back-fill.

**Review by jakobnissen — APPROVED** [review]
Thank you for this work. I only have minor comments.

File: Project.toml [hunk 4]
```diff
@@ -1,24 +1,25 @@
...
 
 [compat]
-julia = "0.7,1"
 BioGenerics = "0.1"
-BioSequences = "2"
-BioSymbols = "4"
+BioSequences = "3"
+BioSymbols = "5"
 IntervalTrees = "1"
+julia = "1"
```
[the remark this answers is not in the document]
Comment (CiaranOMara) [reply Project.toml:18]: https://github.com/BioJulia/BioSequences.jl/commit/3c8f501b0ec4e971433816bd057c11ac33466cb8

File: CHANGELOG.md (hunk 1 above)
Comment (CiaranOMara) [reply CHANGELOG.md:34]: This was resolved by code review.

File: CHANGELOG.md (hunk 2 above)
Comment (CiaranOMara) [reply CHANGELOG.md]: This was resolved by code review.

**Comment by CiaranOMara** [comment]
PR #60 was merged into master and cherry-picked into this branch. 

I think this branch is ready for registration, then merging into master. I'll proceed if no comments to the contrary get raised before 9 am on the 7th of July AEST. Alternatively, if I get an "ok" from all involved, I may be able to proceed earlier.

CC: @MillironX, @jakobnissen, @kescobo, @SabrinaJaye

**Comment by kescobo** [comment]
I trust your judgement on this. Go forth!
